File: objectiveai-js/src/functions/profiles/computations/response/streaming/object.ts
import { z } from "zod";

export const FunctionsProfilesComputationsResponseStreamingObjectSchema = z.literal("function.profile.computation.chunk").meta({ title: "functions.profiles.computations.response.streaming.Object" });
export type FunctionsProfilesComputationsResponseStreamingObject = z.infer<typeof FunctionsProfilesComputationsResponseStreamingObjectSchema>;

File: objectiveai-rs/src/functions/profiles/computations/response/streaming/object.rs
use serde::{Deserialize, Serialize};
use schemars::JsonSchema;

#[derive(Debug, Clone, Copy, Serialize, Deserialize, JsonSchema)]
#[schemars(rename = "functions.profiles.computations.response.streaming.Object")]
pub enum Object {
    #[serde(rename = "function.profile.computation.chunk")]
    FunctionProfileComputationChunk,
}
